EIPS/eip-3290.md

```diff
@@ -0,0 +1,172 @@
+
+Let's list the possible application layer use cases for such a standard: 
+
+1. Lost access to an account
+Users can set up an account and create a switch for it in such a way that, even if he loses his private keys in the future, the funds will be transferred to a predefined secondary account after a period of time.
+
+2. Cryptographic "Testament" or "Last Will."
+Imagine you would like to store your cryptocurrency savings in a contract that will be accessible by members of the family, but only after something has happened to us. Let's say we should "check-in" by evoking some contract function calls every so often.
+If we don't check in on time, something presumably happened to us, and they can withdraw the funds. The proportion of funds they would each receive could either be explicitly set in the contract or be left to decide by consensus among the family members.
+
+3. An insurance fund
+Lock funds and tokens temporarily in a smart contract as a way of insuring them until a certain date.
+
+4. Pension or Savings account
+Another application of locking funds could be to create a small pension fund or time-based savings account, i.e., one that prevents the owner from withdrawing any funds before a certain time in the future. (It could be particularly useful for addicted crypto traders in helping keep their ether intact.)
+
+5. Family trust fund
+Let's imagine we would like to give ether, ERC token, or NFT to someone for their 18th birthday. We could write down on a piece of paper the beneficiary account's private key and the address holding the funds and hand it over to them in an envelope. The only thing they would have to do is call a function on the contract from their account once they are 18, and all the funds will be transferred to them.
+
+6. Vesting
+The most common reason at the moment to lock funds is called "vesting." Imagine that you have just raised a successful ICO and your company still holds a majority of tokens distributed between your team members.
+It would be beneficial to all parties involved to ensure that people's tokens cannot be traded straightaway. If there are no controls in place, anyone might take action by selling all their tokens, cashing out, and quitting the company. This would negatively affect the market price and make all remaining contributors to the project unhappy.
+
+7. DEFI Yielding
+Protect the deposited liquidity or the pool itself.
+
+## Specification
+TBS (Time Based Switch) feature integrates smart contracts and uses events/callbacks to inform the user about the TBS state.
+
+The user creates his TBS by calling the createSwitch function together with four parameters. This returns a signed transaction and puts it in a queue for due execution based on block height or a timestamp.
```

Comment:
This type of content is more appropriate for a tutorial or docs site, separate from the EIP.  The EIP should focus exclusively on defining the requirements of someone wanting to implement the interface, not walking people through how to use it.